Snowdog_Components/components/Organisms/product-grid-item/_product-grid-item-variables.scss

```diff
@@ -75,5 +75,21 @@ $product-grid-item__secondary-action-width--wishlist        : 100% !default;
 $product-grid-item__secondary-action-width--wishlist\@medium: calc(100% - #{$product-grid-item__qty-width--wishlist\@medium}) !default;
 $product-grid-item__secondary-action-width--wishlist\@xl    : calc(100% - #{$product-grid-item__qty-width--wishlist\@xl}) !default;
 
+$product-grid-item__tocompare-background-color              : $gray-lighter !default;
+
 // Slider item variant
 $product-grid-item__image-wrapper-width--slider-item        : 100% !default;
+
+// EE Version
+$product-grid-item-ee__actions-width                        : 100% !default;
+$product-grid-item-ee__tocart-width                         : 48px !default;
+$product-grid-item-ee__tocart-height                        : 48px !default;
+$product-grid-item-ee__requisition-margin                   : 0 $spacer 0 0 !default;
+$product-grid-item-ee__multiwishlist-width                  : 48px !default;
+$product-grid-item-ee__multiwishlist-margin                 : 0 $spacer !default;
+$product-grid-item-ee__dropdown-list-content-width          : 180px !default;
+$product-grid-item-ee__dropdown-list-content-width\@medium  : 210px !default;
+$product-grid-item-slider__dropdown-list-content-top        : auto !default;
+$product-grid-item-slider__dropdown-list-content-bottom     : 100% !default;
+$product-grid-item-ee__dropdown-list-label-width            : 48px !default;
+$product-grid-item-ee__dropdown-list-label-height           : 48px !default;
```

Comment:
missing new line on the end of the file andI think the name of variable should be modified - with modifier:
ex: `$product-grid-item__actions-width--ee` instead of `$product-grid-item-ee__actions-width`